precompiles/parachain-staking/StakingInterface.sol

```diff
@@ -85,7 +85,17 @@ interface ParachainStaking {
     function candidateDelegationCount(address candidate)
         external
         view
-        returns (uint256);
+        returns (uint32);
+
+    /// @dev Get the CandidateAutoCompoundingDelegationCount weight hint
```

Comment:
If we implement refunds correctly when we do weight hints, then it would probably be cheaper to overestimate and get a refund than to call this... :thinking:

Reply:
Probably yeah